dicomanonymizer/simpledicomanonymizer.py

```diff
@@ -238,12 +238,12 @@ def anonymizeDICOMFile(inFile, outFile, extraAnonymizationRules = None):
     """
     dataset = pydicom.dcmread(inFile)
 
-    anonymizeDataset(dataset, extraAnonymizationRules)
+    anonymizeDataset(dataset, extraAnonymizationRules, delete_private_tags)
 
     # Store modified image
     dataset.save_as(outFile)
 
-def anonymizeDataset(dataset, extraAnonymizationRules = None):
+def anonymizeDataset(dataset, extraAnonymizationRules = None, delete_private_tags = True):
```

Comment:
you've got to choose for your variable names: either camel casing or delimiter separating keywords

Reply:
Yes, I'll make a new MR after and will refactoring all my code in order to fit with peep8 convention.